contentcuration/contentcuration/settings.py

```diff
@@ -321,3 +321,6 @@
 # GOOGLE DRIVE SETTINGS
 GOOGLE_AUTH_JSON = "credentials/client_secret.json"
 GOOGLE_STORAGE_REQUEST_SHEET = "16X6zcFK8FS5t5tFaGpnxbWnWTXP88h4ccpSpPbyLeA8"
+
+# Used as the default parent to collect orphan nodes
+GARBAGE_COLLECTION_NODE_ID = "00000000000000000000000000000000"
```

Comment:
id for garbage collection node

Reply:
Just in terms of naming... sounds a bit harsh to throw the poor little things in the bin before they've even had a chance to thrive and find a tree somewhere... Maybe something like "temporary nodes"... or "ORPHANAGE_ROOT"? :D

(not a biggie, just thinking that GC is something that happens on top of it, rather than _being_ it)

Reply:
hahaha yes! I guess calling them garbage right off the bat is a little mean